### PR TITLE
document resuse of vs

### PR DESCRIPTION
**Pull request by kalantar**
@sriumcp @sushmarchandran since you had the recent experience of working with a user who had this problem ... does this help?

### CODE REVIEW

File: content/tasks/vs-reuse.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+menuTitle: Reusing VirtualServices
+title: Reusing VirtualServices
+weight: 30
+summary: Learn how to resuse an existing VirtualService
+---
+To route a percentage of traffic between the baseline and candidate versions of a service, iter8 creates and modifies an Istio `VirtualSystem` and a set of `DestinationRules` (one for each of the baseline and canidate versions).
+In some cases, a `VirtualSystem` may already exist.
```
Comment (sriumcp) [inline content/tasks/vs-reuse.md:8]: Is this 'VirtualSystem' or 'VirtualService'?

Comment (sriumcp) [reply content/tasks/vs-reuse.md:8]: Apart from this, it looks good to me.

**Review by sriumcp — CHANGES_REQUESTED** [review]

**Comment by kalantar** [comment]
I've fixed the names and the link issue was already fixed separately.

**Review by sriumcp — APPROVED** [review]
LGTM.

**Pull request merged**